src/controllers/trust.legal.entity.beneficial.owner.controller.ts

```diff
@@ -16,7 +16,7 @@ type TrustIndividualBeneificalOwnerPageProperties = {
   templateName: string;
   pageData: {
     trustData: CommonTrustData,
-    individualTrusteeType: typeof RoleWithinTrustType;
+    RoleWithinTrustType: typeof RoleWithinTrustType;
```

Comment:
I'm currently making changes to this file in my story. THis being one of them 

` roleWithinTrustType: typeof RoleWithinTrustType;`